src/bindings/js/node/src/node_wrap.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <napi.h>
+#include <ov/node.hpp>
+
+#include <openvino/openvino.hpp>
+
+class NodeWrap : public Napi::ObjectWrap<NodeWrap> {
+public:
+    /**
+     * constructor
+     * @brief Constructs NodeWrap from the Napi::CallbackInfo.
+     * @param info contains passed arguments. Can be empty.
+     * @param node pointer to ov::Node object.
+     */
+    NodeWrap(const Napi::CallbackInfo& info, ov::Node* node);
+
```

Comment:
```suggestion
    NodeWrap() = delete
```
Maybe it should not be allowed to create with default values (nullptr)?